tests/common/fixtures/advanced_reboot.py

```diff
@@ -459,6 +459,24 @@ def imageInstall(self, prebootList=None, inbootList=None, prebootFiles=None):
         # Handle mellanox platform
         self.__handleMellanoxDut()
 
+    def move_logs_before_reboot(self):
+        source_dir = '/host/logs_before_reboot'
+        target_dir = '/var/log'
+
+        command = "test -d {}".format(source_dir)
+        result = self.duthost.shell(command)
+
+        if result["rc"] == 0:
+            command = 'sudo find ' + source_dir + ' -type f -exec sh -c \'mv "$0" "' + target_dir + '/$(basename "$0").preboot"\' {} \\;'
+            result = self.duthost.shell(command)
```

Comment:
Do you want to set module_ignore_errors=True. This is so that the execution of command does not cause unexpected failure/crash of test at this point.

Reply:
Thanks for pointing that out, added

Reply:
Looks like still not added. Am I missing something?

Reply:
I accidentally only added to the shell command above, missed this one. Added